### PR TITLE
DOC: Add questions/contributing section to README

### PR DESCRIPTION
**Pull request by FreddieV4**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -156,10 +150,26 @@ line, run:
...
 resulting performance dataframe to dma.pickle which you can then load
-and analyze from within python.
+and analyze from within Python.
 
 You can find other examples in the ``zipline/examples`` directory.
 
+Questions?
+==========
+
+If you find a bug feel free to `open an issue <https://github.com/quantopian/zipline/issues/new>`_ and fill out the issue template.
```
Comment (richafrank) [inline README.rst]: How about a comma after "bug"?

File: README.rst [hunk 2]
```diff
@@ -156,10 +150,26 @@ line, run:
...
+
+Contributing
+============
+
+All contributions, bug reports, bug fixes, documentation improvements, enhancements and ideas are welcome. Details on how to set up a development environment can be found in our `development guidelines <http://zipline.io/development-guidelines.html>`_
```
Comment (richafrank) [inline README.rst]: Missing a period at the end. Also, I'm a stickler for the oxford comma after "enhancements", if you please.

File: README.rst [hunk 3]
```diff
@@ -156,10 +150,26 @@ line, run:
...
+
+If you are simply looking to start working with the Zipline codebase, navigate to the GitHub `issues` tab and start looking through interesting issues. Sometimes there are of issues labeled as `Beginner Friendly <https://github.com/quantopian/zipline/issues?q=is%3Aissue+is%3Aopen+label%3A%22Beginner+Friendly%22>`_ or `Help Wanted <https://github.com/quantopian/zipline/issues?q=is%3Aissue+is%3Aopen+label%3A%22Help+Wanted%22>`_
```
Comment (richafrank) [inline README.rst]: Let's lose "simply". Extra word "of" in "Sometimes there are of issues".

**Comment by FreddieV4** [comment]
@richafrank good for another pass :)

File: README.rst [hunk 4]
```diff
@@ -156,10 +150,26 @@ line, run:
...
+==========
+
+If you find a bug, feel free to `open an issue <https://github.com/quantopian/zipline/issues/new>`_ and fill out the issue template.
+
+Contributing
+============
+
+All contributions, bug reports, bug fixes, documentation improvements, enhancements, and ideas are welcome. Details on how to set up a development environment can be found in our `development guidelines <http://zipline.io/development-guidelines.html>`_.
+
+If you are looking to start working with the Zipline codebase, navigate to the GitHub `issues` tab and start looking through interesting issues. Sometimes there are issues labeled as `Beginner Friendly <https://github.com/quantopian/zipline/issues?q=is%3Aissue+is%3Aopen+label%3A%22Beginner+Friendly%22>`_ or `Help Wanted <https://github.com/quantopian/zipline/issues?q=is%3Aissue+is%3Aopen+label%3A%22Help+Wanted%22>`_
```
Comment (richafrank) [inline README.rst]: Period missing after the links.

**Pull request merged**